live-build/base/config/package-lists/tools.list.chroot

```diff
@@ -42,6 +42,7 @@ psmisc
 pv
 screen
 sg3-utils
+shellcheck
```

Comment:
Since we're installing this for ZFS development, we should add it to the ZFS Ansible role, [here](https://github.com/delphix/appliance-build/blob/master/live-build/misc/ansible-roles/appliance-build.zfsonlinux-development/tasks/main.yml#L25).

Reply:
Thanks, I was looking for something related to development but couldn't find that.